src/amo/withExperiment.js

```diff
@@ -246,5 +258,5 @@ export const withExperiment = ({
     }
   }
 
-  return withCookies(WithExperiment);
+  return connect()(withCookies(WithExperiment));
```

Comment:
heh, hopefully that addition won't be a problem.

Reply:
It doesn't seem to be. What do you anticipate being a problem with it?